src/gt4sd/domains/materials/__init__.py

```diff
@@ -41,7 +44,13 @@
 Sequence = str
 
 
-def validate_molecules(
+class InputType(str, Enum):
+    selfies = "SELFIES"
+    smiles = "SMILES"
+    copolymer = "COPOLYMER"
```

Comment:
Being even more explicit would be good:
```suggestion
class MoleculeFormat(str, Enum):
    selfies = "SELFIES"
    smiles = "SMILES"
    copolymer = "Copolymer"
```